packages/field-highlighter/src/vaadin-user-tags.js

```diff
@@ -137,7 +141,8 @@ export class UserTags extends PolylitMixin(LitElement) {
 
   /** @protected */
   get wrapper() {
-    return this.$.overlay.querySelector('[part="tags"]');
+    // Used by Collaboration Kit util
+    return this;
```

Comment:
I decided to keep `wrapper` getter to make the following code work:

https://github.com/vaadin/collaboration-kit/blob/cc63f2f409df1dd91d515f981a8a36c2d5aa0f72/collaboration-engine-test/collaboration-engine-test-common/src/main/java/com/vaadin/collaborationengine/util/FieldHighlightUtil.java#L80-L84